gatsby-node.js

```diff
@@ -32,29 +107,46 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     }
   `);
 
-  if (markdownPages.errors || clientPages.errors) {
-    reporter.panicOnBuild(`Error while running GraphQL query.`);
+  if (markdownBlogPages.errors) {
+    reporter.panicOnBuild(`Error while running GraphQL query for blog pages.`);
     return;
   }
 
   // create a page for each markdown file
-  markdownPages.data.allMarkdownRemark.nodes.forEach((node) => {
+  markdownBlogPages.data.allMarkdownRemark.nodes.forEach((node) => {
     createPage({
       path: node.frontmatter.path,
       component: BLOG_POST_TEMPLATE_PATH,
       context: {},
     });
   });
+};
+
+const createClientPages = async ({ actions, reporter, graphql }) => {
```

Comment:
some piggy back refactoring to the client pages in this PR or is it related with the career somehow ? Fine for me in this context 👍